src/triage/component/collate/spacetime.py

```diff
@@ -347,11 +577,23 @@ def get_impute_create(self, impute_cols, nonimpute_cols):
         )
 
         # imputation starts from the state table and left joins into the aggregation table
-        query += "\nFROM %s t1" % self._state_table_sub()
+        query += "\nFROM %s t1" % self._cohort_table_sub()
         query += "\nLEFT JOIN %s t2 USING(%s, %s)" % (
             self.get_table_name(),
-            self.state_group,
+            self.entity_column,
             self.output_date_column,
         )
 
         return "CREATE TABLE %s AS (%s)" % (self.get_table_name(imputed=True), query)
+
+    @property
+    def feature_columns_sans_impflags(self):
+        columns = []
+        for group, groupby in self.groups.items():
+            intervals = self.intervals[group]
+            columns += list(
+                chain(
+                    *[self._get_aggregates_sql(i, "2016-01-01", group) for i in intervals]
```

Comment:
this is what `chain.from_iterable` does:

    chain.from_iterable(self._get_aggregates_sql(i, "2016-01-01", group) for i in intervals)